docs/references/general/dat.md

```diff
@@ -0,0 +1,115 @@
+| g                           | gbr                   | Goose Bay      |
+| s                           | sch                   | Schefferville  |
+| k                           | kap                   | Kapuskasing    |
+| h                           | hal                   | Halley         |
+| t                           | sas                   | Saskatoon      |
+| b                           | pgr                   | Prince George  |
+| a                           | kod                   | Kodiak         |
+| w                           | sto                   | Stokkseyri     |
+| e                           | pyk                   | Pykkvibaer     |
+| f                           | han                   | Hankasalmi     |
+| d                           | san                   | Sanae          |
+| j                           | sys                   | Syowa South    |
+| n                           | sye                   | Syowa East     |
+| r                           | tig                   | TIGER          |
+| p                           | ker                   | Keruelen       |
+| c                           | ksr                   | King Salmon    |
+| u                           | unw                   | Unwin          |
+| i                           | wal                   | Wallops Island |
+
+
+During this time if radars switched modes or stopped mid 2-hour period a new file would be created with a letter after the 1 letter radar abbreviation.
+!!! Warning
+    Do not confuse this with RAWACF letters representing channels
+
+> YYYYMMDDHH.<1-letter radar abbreviation>[a-z].dat
+
+## Fields
+
+DAT files contain a record that contains scalar and vector fields. 
+
+### Scalars
+
+| Field name             | Units     | Data Type | Description                                                                                                                                                                                      |
+| :----------            | :-----:   | :-------: | :---                                                                                                                                                                                             |
+| *radar.revision.major* | *None*    | char      | Major version number of the radar operating system                                                                                                                                               |
+| *radar.revision.minor* | *None*    | char      | Minor version number of the radar operating system                                                                                                                                               |
+| npr                    | *None*    | int       | Number of parameters                                                                                                                                                                             |
```

Comment:
``` suggestion
Total number of scalar and vector fields in a DAT record   
```

I think it's the number of fields or variables.  Since the user can add more variables if needed.